src/components/Select/Select.stories.mdx

```diff
@@ -0,0 +1,214 @@
+
+<ArgsTable story="Select" />
+
+## Labelling Variations
+
+A Select can be rendered with or without visible labels.
+
+<Preview withToolbar>
+  <Story
+    name="Labelling Variations"
+    args={{
+      labelText: "What is your favorite color?",
+      showLabel: false,
+    }}
+  >
+    {args => (
+      <Select {...args}>
+        <option aria-selected={true}></option>
+        <option aria-selected={false}>Red</option>
+        <option aria-selected={false}>Green</option>
+        <option aria-selected={false}>Blue</option>
+        <option aria-selected={false}>Black</option>
+        <option aria-selected={false}>White</option>
+      </Select>
+    )}
+  </Story>
+</Preview>
+
+<Canvas>
+  <Select labelText="What is your favorite color?" showLabel={true}>
+    <option aria-selected={true}></option>
+    <option aria-selected={false}>Red</option>
+    <option aria-selected={false}>Green</option>
+    <option aria-selected={false}>Blue</option>
+    <option aria-selected={false}>Black</option>
+    <option aria-selected={false}>White</option>
+  </Select>
+</Canvas>
+
+<Canvas>
+  <Select
+    labelText="What is your favorite color?"
+    showLabel={false}
+    helperText="Choose wisely!"
+  >
+    <option aria-selected={true}></option>
+    <option aria-selected={false}>Red</option>
+    <option aria-selected={false}>Green</option>
+    <option aria-selected={false}>Blue</option>
+    <option aria-selected={false}>Black</option>
+    <option aria-selected={false}>White</option>
+  </Select>
+</Canvas>
+
+<Canvas>
+  <Select
+    labelText="What is your favorite color?"
+    showLabel={true}
+    helperText="Choose wisely!"
+  >
+    <option aria-selected={true}></option>
+    <option aria-selected={false}>Red</option>
+    <option aria-selected={false}>Green</option>
+    <option aria-selected={false}>Blue</option>
+    <option aria-selected={false}>Black</option>
+    <option aria-selected={false}>White</option>
+  </Select>
+</Canvas>
+
+<Canvas>
+  <Select
+    labelText="What is your favorite color?"
+    showLabel={false}
+    helperText="Choose wisely!"
+    errorText="Waaaaaa!"
+    errored={true}
+  >
+    <option aria-selected={true}></option>
+    <option aria-selected={false}>Red</option>
+    <option aria-selected={false}>Green</option>
+    <option aria-selected={false}>Blue</option>
+    <option aria-selected={false}>Black</option>
+    <option aria-selected={false}>White</option>
+  </Select>
+</Canvas>
+
+<Canvas>
+  <Select
+    labelText="What is your favorite color?"
+    showLabel={true}
+    helperText="Choose wisely!"
+    errorText="Waaaaaa!"
+    errored={true}
+  >
+    <option aria-selected={true}></option>
+    <option aria-selected={false}>Red</option>
+    <option aria-selected={false}>Green</option>
+    <option aria-selected={false}>Blue</option>
+    <option aria-selected={false}>Black</option>
+    <option aria-selected={false}>White</option>
+  </Select>
+</Canvas>
+
+## Deprecated Implementation
+
```

Comment:
Why is this a deprecated implementation? A short explanation would be good to have here.

Reply:
Done.